cmd/enterprise-portal/internal/subscriptionsservice/v1.go

```diff
@@ -46,14 +49,44 @@ func RegisterV1(
 }
 
 type handlerV1 struct {
-	subscriptionsv1connect.UnimplementedSubscriptionsServiceHandler
-
 	logger log.Logger
 	store  StoreV1
 }
 
 var _ subscriptionsv1connect.SubscriptionsServiceHandler = (*handlerV1)(nil)
 
+func (s *handlerV1) GetEnterpriseSubscription(ctx context.Context, req *connect.Request[subscriptionsv1.GetEnterpriseSubscriptionRequest]) (*connect.Response[subscriptionsv1.GetEnterpriseSubscriptionResponse], error) {
+	logger := trace.Logger(ctx, s.logger)
+
+	// 🚨 SECURITY: Require appropriate M2M scope.
+	requiredScope := samsm2m.EnterprisePortalScope("subscription", scopes.ActionRead)
```

Comment:
Throughout this file it seems to use the string literal permission in numerous places rather than using the exported value from the sdk
```suggestion
	requiredScope := samsm2m.EnterprisePortalScope(scopes.PermissionEnterprisePortalSubscription, scopes.ActionRead)
```

Reply:
You're right, this predates the change to make the permissions public consts - updated, thank you!